main/guides/coreeval/permissions.md

```diff
@@ -1,96 +1,55 @@
-* **feeMintAccess**: When this permission is set, the contract will be able to create digital assets. Only contracts that mint privileged Agoric digital assets (i.e., not the unprivileged **[zcf.makeZCFMint()](/reference/zoe-api/zoe-contract-facet.md#zcf-makezcfmint-keyword-assetkind-displayinfo)**) will need access to this.
-* **economicCommitteeCreatorFacet**, **econCharterKit**, **provisionPoolStartResult**: These 3 permissions are required by governed contracts.
-* **chainTimerService**: When this permission is set, it grants access to the *chainTimerService*. All governed contracts need access to this so they know when a vote is complete.
-
-## Top Level Produce Section
+export const main = startSellConcertTicketsContract;
+```
 
-Specifies what, if anything, the contract produces. For example, the example PSM contract 
-produces *testFirstAnchorKit* which is used for testing purposes.
+## Selected BootstrapPowers
 
-## Installation Section 
+In the top level promise space, we have:
 
-Specifies what well-known installations the contract requires. At a minimum, the contract itself should
-be listed as an installation. Governed contracts should include a *contractGovernor* installation.
+- **agoricNames**: read-only access to the [agoricNames](../integration/name-services.md#agoricnames-agoricnamesadmin-well-known-names) name service.
 
-## Instance Section
+- **board**: the [board](../integration/name-services.md#the-board-publishing-under-arbitrary-names) name service.
+  **Note: the board only grows; no mechanism to reclaim storage has been established.**
 
-Specifies what instances, if any, the contract produces or consumes.
+- **chainStorage**: to make storage nodes to write to vstorage.
+  **Warning: this includes the right to over-write previously allocated storage nodes.**
 
-## Issuer Section
+- **chainTimerService**: for getting the current timer and setting timer wake-ups; for example, at the conclusion of a governance vote.
+  **Note: this includes the right to schedule infinitely repeating events.**
 
-Specifies what **[Issuers](/reference/ertp-api/issuer.md)**, if any, the contract produces or consumes.
+- **agoricNamesAdmin**: admin / update access to **agoricNames** and the name hubs it contains.
+  **Warning: this includes the right to over-write existing bindings to instances, brands, etc.**
 
-## Brand Section
+- **bankManager**: to manage reflection of cosmos
+  assets as ERTP assets: to register an issuer to correspond to a denom or to get a bank of purses for any address. **Warning: this includes the right to spend assets for any account.**
 
-Specifies what **[Brands](/reference/ertp-api/brand.md)**, if any, the contract produces or consumes.
\ No newline at end of file
+- **zoe**: the Zoe service.
+  **Note: this includes the right to install any available bundle and start instances of any available installation.**
```

Comment:
starting instances is widely accessible.

Reply:
only if you have access to `zoe`, right?

I suppose every contract can do `zcf.getZoeService()`. hm.